homeassistant/components/sighthound/image_processing.py

```diff
@@ -58,7 +62,11 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     entities = []
     for camera in config[CONF_SOURCE]:
         sighthound = SighthoundEntity(
-            api, camera[CONF_ENTITY_ID], camera.get(CONF_NAME), save_file_folder
+            api,
+            camera[CONF_ENTITY_ID],
+            camera.get(CONF_NAME),
+            save_file_folder,
+            config.get(CONF_SAVE_TIMESTAMPTED_FILE),
```

Comment:
Please use `dict[key]` when we know the key is in the dict.
```suggestion
            config[CONF_SAVE_TIMESTAMPTED_FILE],
```